Cargo.toml

```diff
@@ -1,3 +1,5 @@
+cargo-features = ["profile-rustflags"]
+
```

Comment:
Requires `nightly`. Should we add the following in `.cargo/config.toml`, and remove `console` profile for now?
```
# Enable Tokio's experimental tracing API.
[build]
rustflags = ["--cfg", "tokio_unstable"]
```